### PR TITLE
Add BroadcastReceiver to update screens

### PR DESCRIPTION
**Pull request by jeanpimentel**

### CODE REVIEW

**Comment by jeanpimentel** [comment]
The last commit has mocked content because we need a SQLite to manage downloads-models relationship. I'll start SMTC-1488 tomorrow.

File: app/src/main/java/com/ciandt/dragonfly/example/features/modelselection/ModelSelectionActivity.kt [hunk 1]
```diff
@@ -62,6 +63,11 @@ class ModelSelectionActivity : BaseActivity(), ModelSelectionContract.View {
         presenter.detachView()
```
Comment (ivamluz) [inline app/src/main/java/com/ciandt/dragonfly/example/features/modelselection/ModelSelectionActivity.kt:63]: Detaching view on resume? Is that correct?

Comment (jeanpimentel) [reply app/src/main/java/com/ciandt/dragonfly/example/features/modelselection/ModelSelectionActivity.kt:63]: No, it is at onPause. Expand the code, please. :)

Comment (ivamluz) [reply app/src/main/java/com/ciandt/dragonfly/example/features/modelselection/ModelSelectionActivity.kt:63]: My bad :|

**Review by ivamluz — APPROVED** [review]

**Pull request merged**